### DISCUSSION TITLE
Statamic simple-commerce multi site gives on sub name a 404

### DISCUSSION THREAD

**Discussion by JohanIclicks** [duncanmcclean/simple-commerce#945] [Help] [answered]
After setting it up as a multisite i get on each sub ( ex localhost/fr/ ) a 404 error.

My routes dont show the sub either.
![Screenshot 2023-12-14 at 11 44 04](https://github.com/duncanmcclean/simple-commerce/assets/85666871/58d49500-6a55-47ba-9c04-3885e34eb1f9)

This is my sites.php
![Screenshot 2023-12-14 at 11 44 53](https://github.com/duncanmcclean/simple-commerce/assets/85666871/ce6071e2-7e30-4675-ad54-8deb51ee511e)

**Comment by duncanmcclean** [comment]
Have you tried clearing your route cache? `php artisan route:clear`

**Reply by JohanIclicks** [reply]
Yes tried it but unfortunately.

To make it clear what i get:
![Screenshot 2023-12-14 at 12 55 52](https://github.com/duncanmcclean/simple-commerce/assets/85666871/d73d0cc9-2de0-4473-a036-515569781fb2)

Also without the simple-commerce plugin i can get it to work. So i think i miss something in the settings.
In  the admin the sites are showing and i can switch in there to edit pages.

**Reply by duncanmcclean** [reply]
Hmm, that's weird, have you made any changes to the `routes/web.php` file that the starter kit ships with?

The addon itself shouldn't be doing anything with the frontend routes.

**Reply by JohanIclicks** [reply]
No nothing at all.

My process was:
statamic new mysite duncanmcclean/sc-starter-kit
composer require doublethreedigital/simple-commerce
php please multishop

After that the /french/ gets a 404 on my end.

**Reply by duncanmcclean** [reply]
What does your `config/statamic/sites.php` file look like?

**Reply by JohanIclicks** [reply]
![Screenshot 2023-12-14 at 13 10 45](https://github.com/duncanmcclean/simple-commerce/assets/85666871/ded53c95-8267-4606-8745-9a57c90bf95c)

Added also 'deutsch' to test.

**Reply by duncanmcclean** [reply, marked as answer]
Ah, I think I see why that's happening.

First, check the `APP_URL` in your `.env` is accurate. 

Also, you may want to adjust the URLs of your French & German sites so they use the `APP_URL` value, like this:

```php
'french' => [
  'name' => 'french',
  'locale' => 'en_US',
  'url' => env('APP_URL').'/french',
],
```

Finally, login to the Control Panel and go through any entries (like pages) you want to be available on the other sites and "localize" them (to localize an entry, click into edit it, click on the site in the sidebar with the line through it, then save the entry, it'll then be localized.)

After I did those three things, everything started working for me!
